src/tests/ftest/util/server_utils.py

```diff
@@ -203,6 +207,24 @@ def _prepare_dmg_hostlist(self, hosts=None):
             hosts = self._hosts
         self.dmg.hostlist = hosts
 
+    def _prepare_server_files(self, override=False):
+        """Distribute the server config file and certificates to all server hosts.
+
+        Args:
+            override (bool, optional): force the distribution of the files even if they have already
+                been distributed. Defaults to False.
+        """
+        if not self._files_prepared or override:
+            # Create the daos_server yaml file
+            self.manager.job.temporary_file_hosts = self._hosts.copy()
+            self.manager.job.create_yaml_file(self._external_yaml_data)
```

Comment:
We're NOT running with `--ignore-config`, right? So we DO need the configs?

Reply:
config yml is used after server cleanup.

Reply:
But, just curious, we're not using the `--ignore-config` option anywhere, right?

Reply:
Did a search with "ignore" from ../ftests/util/., we current have macsio and soak test using --ignore-config.

Reply:
I don't see anything using `--ignore-config`. Could you point me to a file/line, please?

Reply:
Sorry from macsio and soak tests are ignore errors, when server start it is required a config yml, --ignore-config could be used for server-stop (cleanup) in the beginning, this used to failure due to no config yml. With the script updates, we prepare the config yml before server-stop (cleanup) in the beginning.

Reply:
Thanks for the explanation